### PR TITLE
feat(content): imprint

### PR DESCRIPTION
**Pull request by ulfgebhardt**
## 🍰 Pullrequest

imprint

### Issues

- None

### Todo

- [X] None

### CODE REVIEW

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -16,6 +16,8 @@ export default defineUserConfig({
     editLink: true,
     lastUpdated: false,
     contributors: false,
+    displayFooter: true,
+    footer: 'CC BY IT4C.dev & Authors. <a href="imprint.html">Imprint</a>.',
```
Comment (mahula) [inline docs/.vuepress/config.js:20]: Just a question:
Does the copyright note require the current year (`© by IT4C.dev & Authors 2023`)?

Comment (ulfgebhardt) [reply docs/.vuepress/config.js:20]: CC BY is creative commons with naming authors https://en.wikipedia.org/wiki/Creative_Commons_license
I thought we do not want to claim any relevant copyright, that's against the philosophy.
The naming of the authors can be removed as well - that then would be `CC0`.

**Review by mahula — DISMISSED** [review]

**Review by Mogge — DISMISSED** [review]

File: docs/.vuepress/config/theme.js [hunk 2]
```diff
@@ -8,7 +8,8 @@ export default hopeTheme({
   editLink: true,
   lastUpdated: false,
   contributors: false,
-  footer: 'CC BY IT4C.dev & Authors. <a href="imprint.html">Imprint</a>.',
+  displayFooter: true,
+  footer: 'CC BY IT4C.dev & Authors. <a href="/imprint.html">Imprint</a>.',
```
Comment (mahula) [inline docs/.vuepress/config/theme.js:12]: Does it need the dot behind the Imprint link?
What about
`footer: 'CC BY IT4C.dev & Authors. <a href="/imprint.html">Imprint</a>',`

Comment (ulfgebhardt) [reply docs/.vuepress/config/theme.js]: image

Comment (mahula) [reply docs/.vuepress/config/theme.js]: OK

**Review by mahula — APPROVED** [review]

**Pull request merged**